examples/node/cli.rs

```diff
@@ -954,40 +955,47 @@ fn list_offers(node: &LightningNode) -> Result<()> {
     Ok(())
 }
 
-fn list_payments(node: &LightningNode, words: &mut dyn Iterator<Item = &str>) -> Result<()> {
+fn list_movements(node: &LightningNode, words: &mut dyn Iterator<Item = &str>) -> Result<()> {
     let number_of_payments: u32 = words
```

Comment:
```suggestion
    let number_of_movements: u32 = words
```

Reply:
Applied in [c94b054](https://github.com/getlipa/lipa-lightning-lib/pull/868/commits/c94b054cbdd4b54ad373a221c9c2edb42ce0adc5)